From 57961c65c2546b28e894adb51f9da0817c825058 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Wed, 22 Feb 2012 14:12:57 +0100
Subject: [PATCH 101/109] scsi: fix wrong return for target INQUIRY

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1329919979-20948-101-git-send-email-pbonzini@redhat.com>
Patchwork-id: 37581
O-Subject: [RHEL 6.3 qemu-kvm PATCH v2 100/102] scsi: fix wrong return for target INQUIRY
Bugzilla: 782029
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>

Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
---
 hw/scsi-bus.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/scsi-bus.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hw/scsi-bus.c b/hw/scsi-bus.c
index 44273c4..83fcda5 100644
--- a/hw/scsi-bus.c
+++ b/hw/scsi-bus.c
@@ -360,7 +360,7 @@ static bool scsi_target_emulate_inquiry(SCSITargetReq *r)
 
     /* PAGE CODE == 0 */
     if (r->req.cmd.xfer < 5) {
-        return -1;
+        return false;
     }
 
     r->len = MIN(r->req.cmd.xfer, 36);
-- 
1.7.7.6

